keps/prod-readiness/sig-auth/2718.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 2718
+stable:
```

Comment:
This is targeting `alpha` not `stable`, correct?

Reply:
Yes.